couch/tests/conftest.py

```diff
@@ -84,82 +115,83 @@ def send_replication(couch_version):
         'create_target': True,
         'continuous': True,
     }
-    r = requests.post(
-        replicator_url,
-        auth=(common.NODE1['user'], common.NODE1['password']),
-        headers={'Content-Type': 'application/json'},
-        json=replication_body,
-    )
-    r.raise_for_status()
-
-
-def get_replication(couch_version):
+    for i in range(5):
+        print("[INFO] create replication task {}".format(i))
```

Comment:
Do we want to keep those print calls? Perhaps we should use a logger instead?

```python
logger = logging.getLogger(__name__)

def dostuff():
    logger.info("...")
```

Reply:
The logger won't work as it and will need to redirect it to stdout (doable, will have a try).

Reply:
Remove `print` and replace with assert messages :)